pymatgen/io/lammps/data.py

```diff
@@ -1373,13 +1375,14 @@ def disassemble(
             for mol in self.mols
         ]
 
+    # TODO(@DanielYang59): should the following two methods be removed?
```

Comment:
Tag: The following two methods do nothing but raise an error. Should they be removed?

Reply:
seems like those methods are needed to override parent class `LammpsData`

https://github.com/materialsproject/pymatgen/blob/ca8f608dec81743d506f365e91efd3bf76d468cf/pymatgen/io/lammps/data.py#L740-L748

Reply:
Thanks for the input. I put a comment there (with your name, because you gave the info) in case some other people ask